server/publications/SimulationPublications.js

```diff
@@ -5,5 +5,11 @@ Meteor.publish('simulations', function() {
         if (user.superuser) {
             return Simulation.find();
         }
+
+        return Simulation.find({_id: user.simulationId});
     }
```

Comment:
You should call `this.ready()` when `this.userId` is undefined otherwise the subscription never returns on the client